tests/test-cases/Group10-VCH-Restart/10-01-VCH-Restart.robot

```diff
@@ -145,7 +149,58 @@ Created Network And Images Persists As Well As Containers Are Discovered With Co
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${alpine}
     Should Be Equal As Integers  ${rc}  0
 
+    Cleanup VIC Appliance On Test Server
+
+Container on Open Network And Port Forwarding Persist After Reboot
+    Set Test Environment Variables
+
+    Log To Console  Create Port Groups For Container network
+    ${out}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run  govc host.portgroup.add -vswitch vSwitchLAN open-net
+
+    ${createcommand}=  catenate  SEPARATOR=\ \
+    ...  bin/vic-machine-linux create --debug 1 --name=%{VCH-NAME}
+    ...  --target=%{TEST_URL}%{TEST_DATACENTER} --thumbprint=%{TEST_THUMBPRINT}
+    ...  --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD}
+    ...  --force=true --bridge-network=bridge --compute-resource=%{TEST_RESOURCE} --no-tlsverify --no-tls
+    ...  --container-network=open-net --container-network-firewall=open-net:open
+
+    ${output}=  Run  ${createcommand}
+
+    # Create a container on the open network
+    ${open-running}=  Launch Container  vch-restart-open-running  open-net
+    ${open-exited}=  Launch Container  vch-restart-open-exited  open-net  ls
+
+    # Create nginx on the open network and check port forwarding
+    Launch Container With Port Forwarding  ${name}=webserver  ${port1}=10000  ${port2}=10001  ${network}=open-net
+
+    # Reboot VCH
+    Reboot VM  %{VCH-NAME}
+    Log To Console  Getting VCH IP ...
+    ${new-vch-ip}=  Get VM IP  %{VCH-NAME}
```

Comment:
This is failure prone in a DHCP environment as the IP address we get is likely to be from the prior boot. Keyword `Wait For VCH Initialization` specifically to avoid this class of error (added for Nimbus HA testing).
Assuming the standard variable names are used and are therefore updated by this keyword, we would have :
```
Reboot VM  %{VCH-NAME}
Wait For VCH Initialization  20x  5 seconds

# Check if the open container is persisted
```